.github/workflows/build-test-base.tmp.yml

```diff
@@ -0,0 +1,189 @@
+################################################################################################################################################################
```

Comment:
Ignore this file, won't be merged in. Copied it over from base for proof of concept.